composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "doofinder/doofinder-magento2",
-    "version": "0.6.2",
+    "version": "0.6.3",
```

Comment:
Now that I'm thinking about it, we should put the version in 0.7.0 because I think it's an important change, not a minor one. What do you think?